pool/pool.go

```diff
@@ -103,10 +85,9 @@ func (p *Pool) ReorgTx(ctx context.Context, tx types.Transaction) error {
 	// Execute transaction to calculate its zkCounters
 	zkCounters, err := p.PreExecuteTx(ctx, tx)
 	if err != nil {
-		log.Warnf("ignoring error while preexecuting tx: %+v", poolTx)
+		poolTx.ZKCounters = zkCounters
```

Comment:
Is it ok to continue if the `err != nil`?

Reply:
Yes! We try our best to get the proper zkCounters but the state may not be ready yet to execute this Tx. Imagine a sc deployment followed by a call to that SC. If the pre-execution of the call happens before the SC is stored in the state, it will fail in pre-execution, but not on real execution. We won't have quality zk counters, but we must process that tx.